packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -282,10 +307,12 @@ export const LocationDetailViewTable = (): JSX.Element => {
                     }}
                     key={`${index}-${row.key}`}
                   >
-                    {row.key}
+                    <TableDataIcon variant="folder" /> {row.key}
                   </TableDataButton>
                 ) : (
-                  <>{parseTableData(row, column)}</>
+                  <TableDataContainer>
```

Comment:
Why do we need this?

Reply:
This is to provide a way for there to be consistent spacing/padding in the table. Some table cells have buttons in them which have their own padding. So without *something* to apply padding to table cells that just have text in them (like a file name), the text doesn't line up with cells that have a button, like here Where the folder names look indented):

<img width="246" alt="Screenshot 2024-08-14 at 2 14 12 PM" src="https://github.com/user-attachments/assets/1f6ba1c9-c1f1-4c94-8221-bf542d694b96">

or here (where text doesn't line up to table headers, because they have sort buttons):

<img width="104" alt="Screenshot 2024-08-14 at 2 14 21 PM" src="https://github.com/user-attachments/assets/26ab40a0-b43a-47d6-8900-77df5c490734">

We'll probably also need this wrapper to do some text truncation also.

Reply:
It could also just be a base View element? It really just needs a `<div>` with a class on it.

Reply:
Hmmm, should we be able to use flex positioning on the `TableRow` and `TableCell` components to center content vertically? It feels off to me that the padding of the `button` is forcing changes to parent HTML structure

Reply:
It won't be just the vertical height that is different, but also the horizontal space without it (like this screenshot with no wrapper element to provide padding to the text elements):

<img width="1118" alt="Screenshot 2024-08-14 at 2 43 01 PM" src="https://github.com/user-attachments/assets/4aaed6df-e72c-43b1-94f9-e542737b0df7">

I could use :has() in css to remove excess padding if the td has a button in it, but I think we'll need a wrapper for text truncation anyways.